mobile/app/(app)/change-password.tsx

```diff
@@ -133,7 +132,11 @@ const ChangePassword = () => {
           control={control}
           render={({ field: { onChange, value } }) => (
             <PasswordInput
-              error={errors.currentPassword?.message}
+              error={
+                errors.currentPassword?.message || reqError
```

Comment:
i don't think we need the `reqError` anymore, we can remove the state and only pass this message